content/en/docs/concepts/services-networking/connect-applications-service.md

```diff
@@ -134,6 +134,10 @@ about the [service proxy](/docs/concepts/services-networking/service/#virtual-ip
 Kubernetes supports 2 primary modes of finding a Service - environment variables
 and DNS. The former works out of the box while the latter requires the
 [CoreDNS cluster addon](http://releases.k8s.io/{{< param "githubbranch" >}}/cluster/addons/dns/coredns).
+{{< note >}}
+Environment variables mode can be disabled using the `enableServiceLinks` flag on the [pod spec](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#pod-v1-core). 
```

Comment:
This is a little vague.  Do you disable the environments variable mode  by setting enableServiceLinks flag to true or false?  You should be clear on how this value should be set to disable the environment variables mode.  You may also want to motivate why this is needed. For example, mention that service discovery environment variables clashing with env vars can result in hard to debug issues and setting this flag properly can help debug those situations

Reply:
Thanks for your input. I was hesitant to add too much information to this document. I will follow your recommendation.